sound/soc/codecs/rt711-sdw.c

```diff
@@ -144,6 +144,9 @@ static int rt711_update_status(struct sdw_slave *slave,
 	/* Update the status */
 	rt711->status = status;
```

Comment:
@shumingfan Please make one patch per logical change and for each patch please explain the reason why the patch is needed. 

Also, please use git log  sound/soc/codecs/rt711-sdw.c and notice the style of the commit subject.

So, in your case you will have 3 patches:

1) ASoC: codecs: rt711: Simplify return value

ret is always 0 and never changed so directly return 0 in order to remove one variable
from stack and also make code more readable.

2) 3) in the same manner.